test/e2e/framework/framework.go

```diff
@@ -270,7 +270,7 @@ func UpdateDeployment(kubeClientSet kubernetes.Interface, namespace string, name
 		return err
 	}
 
-	err = WaitForPodsReady(kubeClientSet, 60*time.Second, replicas, namespace, metav1.ListOptions{
+	err = WaitForPodsReady(kubeClientSet, 90*time.Second, replicas, namespace, metav1.ListOptions{
```

Comment:
was flaky?

Reply:
yep :/

Reply:
probably started to be an issue with 60s because you're spinning up 30 replicas

Reply:
haha yeah, I'm pretty sure that's it.